Sources/tart/VMStorageOCI.swift

```diff
@@ -200,8 +200,18 @@ class VMStorageOCI: PrunableStorage {
           // Choose the best base image which has the most deduplication ratio
           let localLayerCache = try await chooseLocalLayerCache(name, manifest, registry)
 
+          if let llc = localLayerCache {
```

Comment:
Let's add a check that `llc.deduplicatedBytes` is more than 0.

Reply:
Added a check in https://github.com/cirruslabs/tart/pull/864/commits/a053c6b981202a703beb6c5c1804b94d28c1534b.